### ISSUE TITLE
Support page actions

### ISSUE THREAD

**Issue by ralfstx** [eclipsesource/tabris-js#8]
It should be possible to bind actions to a page, so that these actions are only visible while the page is active.

**Comment by ralfstx** [comment]
With the page events in 0.9.0, it's already possible to display an action on certain pages only:

```js
page.on("appear", function() {
  action.set("visible", true);
}).on("disappear", function() {
  action.set("visible", false);
});
```

That's probably not the perfect solution, but it reduces the urgency of a "page actions" feature.

**Comment by mpost** [comment]
I would still think it is very convenient to bind an action directly to a page as requested in #329

**Comment by sand123** [comment]
I agree. Action constructor should accept one more parameter like ```pageSelector``` to match ```tabris.Page``` instances

**Comment by tbuschto** [comment]
Could be implemented something like this: https://github.com/dbuschtoens/reddit_viewer/blob/master/src/ui/ViewModeToggleAction.ts (but maybe using visibility instead of attach/detach)
